web/src/views/User.vue

```diff
@@ -27,48 +29,61 @@
   </FluidContainer>
 </template>
 
-<script lang="ts">
-import { computed, defineComponent, onMounted, ref } from 'vue';
+<script lang="ts" setup>
+import { useLocalStorage } from '@vueuse/core';
+import dayjs from 'dayjs';
+import TimeAgo from 'javascript-time-ago';
+import { computed, onMounted, ref } from 'vue';
 import { useI18n } from 'vue-i18n';
 
 import Button from '~/components/atomic/Button.vue';
+import SelectField from '~/components/form/SelectField.vue';
 import FluidContainer from '~/components/layout/FluidContainer.vue';
 import useApiClient from '~/compositions/useApiClient';
 
-export default defineComponent({
-  name: 'User',
+const { t, availableLocales, locale } = useI18n();
 
-  components: {
-    FluidContainer,
-    Button,
-  },
+const apiClient = useApiClient();
+const token = ref<string | undefined>();
 
-  setup() {
-    const apiClient = useApiClient();
-    const token = ref<string | undefined>();
+onMounted(async () => {
+  token.value = await apiClient.getToken();
+});
 
-    onMounted(async () => {
-      token.value = await apiClient.getToken();
-    });
+// eslint-disable-next-line no-restricted-globals
+const address = `${location.protocol}//${location.host}`;
```

Comment:
address should also have `${location.port}`

Reply:
incase its not 443

Reply:
Description from `lib.dom.d.ts`:

```ts
    /**
     * Returns the Location object's URL's host and port (if different from the default port for the scheme).
     *
     * Can be set, to navigate to the same URL with a changed host and port.
     */
    host: string;
```

Reply:
In that case add a comment please.

Reply:
status ?!?

Reply:
Added a comment.